tools/generator/Java.Interop.Tools.Generator.ObjectModel/Field.cs

```diff
@@ -52,54 +68,5 @@ public abstract class Field : ApiVersionsSupport.IApiAvailability {
 			// within the api.xml description.
 			"java.lang.String",
 		};
-		
-		ISymbol symbol;
-
-		ParameterList setParameters;
-
-		internal string GetMethodPrefix {
-			get { return (Symbol is SimpleSymbol || Symbol.IsEnum) ? StringRocks.MemberToPascalCase (Symbol.JavaName) : "Object"; }
-		}
-
-		public bool IsConst {
-			get { return IsFinal && IsStatic; }
-		}
-
-		public bool NeedsProperty {
-			get { return !IsStatic || !IsFinal || String.IsNullOrEmpty (Value) || Symbol.IsArray || !primitive_types.Contains (Symbol.JavaName); }
-		}
-
-		public ISymbol Symbol {
-			get { return symbol; }
-		}
-
-		internal string ID {
-			get { return JavaName + "_jfieldId"; }
-		}
-
-		internal ParameterList SetParameters {
-			get { return setParameters; }
-		}
-
-		public string Annotation { get; internal set; }
-
-		public bool Validate (CodeGenerationOptions opt, GenericParameterDefinitionList type_params, CodeGeneratorContext context)
```

Comment:
This change confuses me; GitHub says "`Field.Validate()` is removed", yet when I view the updated file within this PR, it's there!

https://github.com/xamarin/java.interop/blob/62c1112c55dc412f94ef90f4bdb37c0631fd6900/tools/generator/Java.Interop.Tools.Generator.ObjectModel/Field.cs#L34

I think this is another example in which reordering the members is creating confusion.  In this situation, I had to really look to find the "new" `Validate()` method declaration.

Did the members need to be re-ordered?

Reply:
They did not need to be, it was more an artifact of how I made these changes.  I made a new class and copied each method over to the it, making the necessary changes as I went.  Then at the end I deleted the original class and renamed mine.

I struggled with how to order the methods.  Generally auto-properties tend to be at the top since they're more like fields, and I was changing a lot of property overrides to auto-properties so that was causing a lot of movement.  Additionally there was no discernible pattern to the method order for me to emulate.  Ultimately I decided to alphabetize them.

I can un-sort them if needed, but that's not a trivial task.  :)